packages/airnode-validator/src/ois/ois.ts

```diff
@@ -195,7 +196,7 @@ export const endpointSchema = z.object({
   summary: z.string().optional(),
 });
 
-const ensureSingleParameterUsagePerEndpoint: ValidatorRefinement<OIS> = (ois, ctx) => {
+const ensureSingleParameterUsagePerEndpoint: ValidatorRefinement<SchemaType<typeof baseOisSchema>> = (ois, ctx) => {
```

Comment:
Nit: could  extract `SchemaType<typeof baseOisSchema>`  and import it from `types.ts`

Reply:
Yeah, definitely, but I think it's better to do it all at once as part of this task: https://api3dao.atlassian.net/browse/AN-723